tests/LoggerTest.php

```diff
@@ -175,10 +175,16 @@ public function testAdapters()
         $response = $logger->addLog($log);
         $this->assertEquals(202, $response);
 
-         // Test LogOwl
-         $adapter = new LogOwl(\getenv("TEST_LOGOWL_KEY"));
-         $logger = new Logger($adapter);
-         $response = $logger->addLog($log);
-         $this->assertEquals(200, $response);
+        // Test LogOwl
+        $adapter = new LogOwl(\getenv("TEST_LOGOWL_KEY"));
+        $logger = new Logger($adapter);
+        $response = $logger->addLog($log);
+        $this->assertEquals(200, $response);
+
+        // Test HoneyBadger
+        $adapter = new HoneyBadger(\getenv("TEST_HONEYBADGER_KEY"));
+        $logger = new Logger($adapter);
+        $response = $logger->addLog($log);
```

Comment:
I tried to test this and the tests say it passed, but I don't see anything in honeybadger. 

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/1477010/202332307-6eb10cd9-3e83-4c6a-b59f-1c4af3fe5594.png">

Any idea what I'm doing wrong?

Reply:
This has been fixed now. Looks like it gives 201 status code even if you give the wrong body. They don't check much at the backend